applications/byte_patrol/tests.py

```
@@ -1,2 +1,377 @@
+        logger.debug(f"\nProcessing request with CustomClientHandler: {request.path}")
+        logger.debug(f"Request META: {request.META}")
+
+        # Call each middleware in the chain directly to debug
+        logger.debug("Calling each middleware individually:")
+        for middleware in self._middleware_chain:
+            logger.debug(f"  Calling {middleware.__class__.__name__}")
+            try:
+                # Try calling middleware directly
+                response = middleware(request)
+                logger.debug(
+                    f"  Response from {middleware.__class__.__name__}: {response.status_code if response else 'None'}"
```

Pylint (3.3.7)

[convention] Line too long (118/100)

(C0301)